client/src/PlayerLanding/OrderPlayers.tsx

```diff
@@ -0,0 +1,54 @@
+import { Button, Flex, Grid } from "@radix-ui/themes";
+import React from "react";
+import { useOrderPlayer, usePlayerNamesToRoles } from "../store/useStore";
+import "./OrderPlayers.css";
+
+interface OrderPlayersProps {
+  myName: string;
+}
+
+export function OrderPlayers({ myName }: OrderPlayersProps) {
+  const players = Object.values(usePlayerNamesToRoles())
+    .map(({ name }) => name)
+    .filter((name) => name !== myName);
+  const [neighbors, setNeighbors] = React.useState<string[]>(["", ""]);
+  const [, , , handleOrderPlayer] = useOrderPlayer();
+
+  return (
+    <Flex direction="column" align="center" gap="4">
+      <Grid columns={"3"} gap="2">
+        {players.sort().map((pl) => (
+          <Button
+            className="name-button"
+            key={pl}
+            disabled={neighbors.includes(pl)}
+            onClick={() => {
+              const newNeighbors = [...neighbors, pl].slice(1);
```

Comment:
this is kinda weird, I would do the opposite.  Add the to end, and slice 0,2 otherwise neighbors could get to 3,4,5 length